src/Symfony/Component/DependencyInjection/Alias.php

```diff
@@ -11,17 +11,24 @@
 
 namespace Symfony\Component\DependencyInjection;
 
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+
 class Alias
 {
     private $id;
     private $public;
     private $private;
+    private $deprecated;
+    private $deprecationTemplate;
+
+    private static $defaultDeprecationTemplate = 'The "%service_id%" service alias is deprecated. You should stop using it, as it will soon be removed.';
```

Comment:
I know this PR is merged, but I've just reviewed its code and I wondered if the `%service_id%` placeholder name is confusing to other people too. To me, this placeholder would be easier to understand if it was called `%alias_id%`.

Also, this -> `... as it will soon be removed.` could be -> `... as it will be removed in the future.`

Thanks.

Reply:
updated in #29995, please review